project_key/models/project_task.py

```diff
@@ -46,7 +47,13 @@ def create(self, vals):
 
         if project_id:
             project = self.env['project.project'].browse(project_id)
-            vals['key'] = project.get_next_task_key()
+            key = vals.get('key', False)
+            if key:
+                seq = int(key.split("-")[1])
```

Comment:
Keeping `"` and `'` consistent would be great!

Reply:
Yes, that is good advice. I am going to update it.